site/about/glossary/_mrm.qmd

```diff
@@ -0,0 +1,44 @@
+#### Model risk management
+
+1st line of defense
+: Business unit(s) responsible for model development, initial validation, and implementation during the model lifecycle. As the 1st line of defense, model developers must document and test models to ensure that they are accurate, robust, and fit for purpose.
+
+2nd line of defense
+: An independent oversight function that provides a governance framework for the model lifecycle. As the 2nd line of defense, model validators must independently validate and challenge models created by model developers to ensure that model risk management principles are followed. 
+
+3rd line of defense
+: Typically an internal audit function responsible for providing an independent and comprehensive review of the risk management processes and controls that the first two lines have implemented. 
+
+model developer
+: Responsible for the design, implementation, and maintenance of models to ensure they are fit-for-purpose, accurate, and aligned with business requirements. As subject matter experts, they collaborate with model validators and other business units, ensuring the models are conceptually sound and robust. 
+
+model governance
+: A framework of policies, procedures, and standards established to oversee the lifecycle of models within an organization. Ensures that models are developed, validated, implemented, and retired in a controlled and consistent manner, promoting accountability, transparency, and adherence to regulatory requirements.
+
+model implementation
+: A collaborative effort among model developers, model owners, and model users. Model implementation includes a formalized implementation plan and associated procedures, a review of results, and a record of model change procedures. 
+
+model owner
+: Responsible for coordinating model development, model implementation, ongoing model monitoring and maintaining the model’s administration (e.g. model documentation and model risk reporting).
```

Comment:
> the model’s administration (e.g. model documentation and model risk reporting)

Could you maybe break up this sentence, avoid the brackets and make the second sentence more about lifecycle management of the model? As is, this read slightly oddly, though I couldn't quite tell you why.

Reply:
```suggestion
: Responsible for coordinating model development, model implementation, ongoing model monitoring and maintaining the model’s administration, such as model documentation and model risk reporting.
```